developer_guide/content/sections/sqs.md

```diff
@@ -0,0 +1,29 @@
+
+# Establish connection with your SQS 
+
+* be sure your user have proper permissions
+
+```ruby
+    aws_access_key_id =  'AxxxxxxxxxxxxxxxA'
```

Comment:
I'm ok with that, will change this in next commit.  ...please let me know if you want me to squash commits